mpas_analysis/config.default

```diff
@@ -296,14 +296,14 @@ plotTitles = ['Arctic', 'Equatorial (15S-15N)', 'Southern Ocean', 'Nino 3',
 ## modules
 
 # title and axis font properties for single-panel plots
-titleFontSize = 20
+titleFontSize = 16
 titleFontColor = black
 titleFontWeight = normal
-axisFontSize = 16
+axisFontSize = 12
 
 # title and axis font properties for three-panel plots
-threePanelTitleFontSize = 20
-threePanelPlotTitleFontSize = 16
+threePanelTitleFontSize = 16
+threePanelPlotTitleFontSize = 14
```

Comment:
@milenaveneziani, these are the main changes I though you might be concerned about.

I have been feeling for quite a while that the titles are too big compared to the other text.  It is helpful with the thumbnails but it makes things look pretty bad at full size and it's starting to creep into our publications.  Also, this would make our plots at least somewhat more consistent with E3SM_Diags, which has panel titles that are the same font size as the axes.